### DISCUSSION TITLE
You are marked as spam, and therefore cannot authorize a third party application. ?

### DISCUSSION THREAD

**Discussion by Sazy67** [community/community#114650] [New to GitHub]
![image](https://github.com/community/community/assets/116670693/cc7a2bee-7e33-4450-add1-bf35b148e732)

Hello, how can I solve this problem in my account? can anyone help? TR.

**Comment by benogoulart** [comment]
Hi,

The message **"You are marked as spam, and therefore cannot authorize a third-party application."** usually means GitHub's automated systems have temporarily flagged your account to prevent abuse.

Unfortunately, this isn't something that can be resolved by the community.

Here are a few things you can do:

- Check your email for any messages from GitHub regarding account restrictions.
- Make sure your account information is complete and verified (such as your email address).
- Avoid creating multiple accounts or performing actions that could trigger GitHub's spam detection.
- Contact GitHub Support if you believe your account was flagged by mistake. They are the only ones who can review and remove account restrictions.

If you're comfortable sharing, could you let us know:
- Is this a brand-new account?
- Does the warning appear when authorizing every third-party application, or only a specific one?
